Repeat/r1.py

```diff
@@ -0,0 +1,43 @@
+
+from PIL import Image
+import numpy as np     # numpy
+from collections import namedtuple
+
+_RepeatBase = namedtuple('repeat', ['count','contents'])
+
+class Repeat(_RepeatBase):
+	"""Repeat class inherits from the _RepeatBase namedtuple"""
```

Comment:
Comments should be about why something exists, not just repeating exactly what the code says.